### PR TITLE
Add semantic convention of influxDB

### PR DESCRIPTION
**Pull request by steverao**
## Changes

Please provide a brief description of the changes here.
Add semantic convention of influxDB. Relative to  https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10850

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [x] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: model/trace/database.yaml [hunk 1]
```diff
@@ -251,3 +251,24 @@ groups:
...
           conditionally_required: when available
         tag: tech-specific
+
+  - id: db.influxdb
+    type: span
+    extends: trace.db.common
+    brief: >
+      Attributes for InfluxDB
+    attributes:
+      - ref: db.name
```
Comment (AlexanderWert) [inline model/trace/database.yaml:261]: Note: `db.name` is being changed in another PR: https://github.com/open-telemetry/semantic-conventions/pull/911

File: .chloggen/influxdb.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
```
Comment (trisch-me) [inline .chloggen/influxdb.yaml:7]: I think new component is used if you create brand new type in registry and in this case  it should be enhancement

@joaopgrassi could you confirm if it's correct?

**Comment by lmolkova** [comment]
@steverao  do you envision adding any specifics about InfluxDB? Like custom attributes, metrics, some non-obvious mapping for common attributes?

If the goal is to add a constant for `db.system`, then it can be done without adding `influxdb.md` or the yaml section for it.
Curious what you think and what's your motivation to add things beyond system constant.
Thanks!

File: .chloggen/influxdb.yaml (hunk 2 above)
Comment (steverao) [reply .chloggen/influxdb.yaml:7]: > I think new component is used if you create brand new type in registry and in this case it should be enhancement
> 
> @joaopgrassi could you confirm if it's correct?

I finished the changelog according to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#adding-a-changelog-entry
It doesn't look like it contains the distinction between `new_component` and `enhancement`. At first, I though this is a new framework so choose `new_component`. If we discuss the difference between them this time, I think it had better to add some explanation about them in document later：）

**Comment by steverao** [comment]
> @steverao do you envision adding any specifics about InfluxDB? Like custom attributes, metrics, some non-obvious mapping for common attributes?

At present, besides `db.system`, I also added some information about new database framework of InfluxDB, such as attributes and examples in https://github.com/open-telemetry/semantic-conventions/pull/949/files#diff-b0b92e2b139407d14dd05fec354af626034d748c5ced214a074e1e2389cffc2c

**Comment by steverao** [comment]
> it does not look like influxDB needs any customization on top of generic attributes and descriptions we have for common DB attributes/spans. Am I correct?

Yes, for the plugin, it don't contain special customization.

> There is no requirement to add a semconv file for each DB.

At first, I saw other database plugins provide relevant YAML document, so I added it for InfluxDB. For this situation, don' you recommend to add its unique YAML document? In the document, I added some explanations for its attributes. I think it  will be better if we conserve it. If you think it's not necessary. I can delete it later.

File: .chloggen/influxdb.yaml (hunk 2 above)
Comment (steverao) [reply .chloggen/influxdb.yaml]: > new_component is a new namespace, in this case this is extension of existing db.

Got it, I modified it now.

**Comment by steverao** [comment]
> It seems that InfluxDB usage of common attributes is very well aligned with common definitions (database name is

Thank you for your explanation! I deleted other contents except `db.system` constant in InfluxDB YAML document now. Please help me to review whether there are any other problems again?

**Comment by steverao** [comment]
It seems there is unstable URL in other document which caused the CI running failed!
image

File: docs/database/influxdb.md
Comment (lmolkova) [inline docs/database/influxdb.md:1]: There is no need to add the file since it does not provide any additional information

Comment (steverao) [reply docs/database/influxdb.md:1]: > There is no need to add the file since it does not provide any additional information

done

**Review by trask — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Pull request merged**